<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hang</class>
 <widget class="QWidget" name="hang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #696969;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>771</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="1" column="3">
     <widget class="QPushButton" name="leave_hang">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>离客厅</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/离房.png</normaloff>:/icons/离房.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="air_equality">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>空气质量</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/空气质量检测.png</normaloff>:/icons/空气质量检测.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="air_condition">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>空调</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/空调.png</normaloff>:/icons/空调.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="hang_light">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>客厅灯</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/客厅灯.png</normaloff>:/icons/客厅灯.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="back_hang">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>回客厅</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/回房.png</normaloff>:/icons/回房.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="read">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>音乐</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/音乐.png</normaloff>:/icons/音乐.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="monitor">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>监控</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/监控.png</normaloff>:/icons/监控.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="dec_light">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>装饰灯</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/装饰灯.png</normaloff>:/icons/装饰灯.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
}</string>
   </property>
   <property name="text">
    <string>2025-03-08 19:22:01</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="position_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
}</string>
   </property>
   <property name="text">
    <string>         客厅</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>position_label</zorder>
  <zorder>time_label</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
